README.md

```diff
@@ -1,16 +1,12 @@
 # Action for Running MATLAB Commands
 
-The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, or statements on a [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) or [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner:
-
-- To use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
-
-- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to set up MATLAB on the runner. Currently, this action is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, and statements on a self-hosted or GitHub&reg;-hosted runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
 
 ## Examples
 Use the **Run MATLAB Command** action to run MATLAB scripts, functions, and statements. You can use this action to flexibly customize your test run or add a step in MATLAB to your workflow.
 
 ### Run MATLAB Script on Self-Hosted Runner
-Use a self-hosted runner to run the commands in a file named `myscript.m` in the root of your repository.
+Use a [self-hosted runner](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) to run the commands in a file named `myscript.m` in the root of your repository.
```

Comment:
It would be nice to get the "R2013b or later" back in here somewhere.

Reply:
Decided to leave it out after discussing with Mark. Will add the information if we have customer requests.